lib/etcunpack.cxx

```diff
@@ -22,256 +22,247 @@
+                                 int channels);
+extern void decompressBlockETC21BitAlphaC(uint block_part1, uint block_part2, uint8* img, uint8* alphaimg, int width, int height,
+                                          int startx, int starty, int channels);
+extern void decompressBlockAlphaC(uint8* data, uint8* img, int width, int height, int startx, int starty, int channels);
+extern void decompressBlockAlpha16bitC(uint8* data, uint8* img, int width, int height, int startx, int starty, int channels);
 
 extern void setupAlphaTable();
 
 // This global variable affects the behaviour of decompressBlockAlpha16bitC.
 extern int formatSigned;
 
 static void
-readBigEndian4byteWord(ktx_uint32_t* pBlock, const GLubyte *s)
-{
-	*pBlock = (s[0] << 24) | (s[1] << 16) | (s[2] << 8) | s[3];
+readBigEndian4byteWord(ktx_uint32_t* pBlock, const GLubyte* s) {
+    *pBlock = (s[0] << 24) | (s[1] << 16) | (s[2] << 8) | s[3];
 }
 
-
 /* Unpack an ETC1_RGB8_OES format compressed texture */
 extern "C" KTX_error_code
-_ktxUnpackETC(const GLubyte* srcETC, const GLenum srcFormat,
-			  ktx_uint32_t activeWidth, ktx_uint32_t activeHeight,
-			  GLubyte** dstImage,
-			  GLenum* format, GLenum* internalFormat, GLenum* type,
-			  GLint R16Formats, GLboolean supportsSRGB)
-{
-	unsigned int width, height;
-	unsigned int block_part1, block_part2;
-	unsigned int x, y;
-	/*const*/ GLubyte* src = (GLubyte*)srcETC;
-	// AF_11BIT is used to compress R11 & RG11 though its not alpha data.
-	enum {AF_NONE, AF_1BIT, AF_8BIT, AF_11BIT} alphaFormat = AF_NONE;
-	int dstChannels, dstChannelBytes;
-
-	switch (srcFormat) {
-	  case GL_COMPRESSED_SIGNED_R11_EAC:
-		if (R16Formats & _KTX_R16_FORMATS_SNORM) {
-			dstChannelBytes = sizeof(GLshort);
-			dstChannels = 1;
-			formatSigned = GL_TRUE;
-			*internalFormat = GL_R16_SNORM;
-			*format = GL_RED;
-			*type = GL_SHORT;
-			alphaFormat = AF_11BIT;
-		} else
-			return KTX_UNSUPPORTED_TEXTURE_TYPE; 
-		break;
-
-	  case GL_COMPRESSED_R11_EAC:
-		if (R16Formats & _KTX_R16_FORMATS_NORM) {
-			dstChannelBytes = sizeof(GLshort);
-			dstChannels = 1;
-			formatSigned = GL_FALSE;
-			*internalFormat = GL_R16;
-			*format = GL_RED;
-			*type = GL_UNSIGNED_SHORT;
-			alphaFormat = AF_11BIT;
-		} else
-			return KTX_UNSUPPORTED_TEXTURE_TYPE; 
+_ktxUnpackETC(const GLubyte* srcETC, const GLenum srcFormat, ktx_uint32_t activeWidth, ktx_uint32_t activeHeight,
+              GLubyte** dstImage, GLenum* format, GLenum* internalFormat, GLenum* type, GLint R16Formats, GLboolean supportsSRGB) {
+    unsigned int width, height;
+    unsigned int block_part1, block_part2;
+    unsigned int x, y;
+    /*const*/ GLubyte* src = (GLubyte*)srcETC;
+    // AF_11BIT is used to compress R11 & RG11 though its not alpha data.
+    enum { AF_NONE, AF_1BIT, AF_8BIT, AF_11BIT } alphaFormat = AF_NONE;
+    int dstChannels, dstChannelBytes;
+
+    switch (srcFormat) {
+    case GL_COMPRESSED_SIGNED_R11_EAC:
```

Comment:
Can you indent the cases by 2 spaces? Since I'm always fighting editors to do this we probably shouldn't continue but I'm curious.